events/api.py

```diff
@@ -2511,13 +2511,39 @@ def _get_queryset_from_cache_many(params, param, cache_name, operator, queryset)
     return queryset
 
 
+def _param_to_keywords(param: str):
+    """
+    Convert comma separated param str into keyword objects with
+    replacements applied.
+    """
```

Comment:
I think it would make more sense to have this take a list of keyword ids instead of a very specific string format, and call it `find_keyword_replacements` or something. Could also split this into two functions, `find_keyword_replacements` and `_param_to_keywords`, where the latter would call the former.

This is more of a matter of opinion than anything actually important, so feel free to leave it as it is; I just like having my functions with as little responsibility as possible.

Reply:
Addressed in c3f71d1d44f1b551acf3dd5491c613547bcd9141

Renamed and changed to accept a list of keyword ids